azext_edge/edge/commands_edge.py

```diff
@@ -61,9 +61,11 @@ def check(
     load_config_context(context_name=context_name)
     from .providers.checks import run_checks
 
-    # by default - run prechecks if AIO is not deployed
-    run_pre = not ORC_API_V1.is_deployed() if pre_deployment_checks is None else pre_deployment_checks
-    run_post = True if post_deployment_checks is None else post_deployment_checks
+    aio_deployed = MQ_ACTIVE_API.is_deployed()
```

Comment:
can we use meta? or would mq still be fine (since it will be there)

Reply:
I just needed something non-orc, i'm sure meta would be fine too but this just unblocks us for now

Reply:
lol got the same place edited using meta because I'm removing orc😆

Reply:
meta is probably the most appropriate